x-pack/plugins/graph/kibana.json

```diff
@@ -4,12 +4,29 @@
   "kibanaVersion": "kibana",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["licensing", "data", "navigation", "savedObjects", "kibanaLegacy"],
-  "optionalPlugins": ["home", "features"],
-  "configPath": ["xpack", "graph"],
-  "requiredBundles": ["kibanaUtils", "kibanaReact", "home"],
+  "requiredPlugins": [
+    "licensing",
+    "data",
+    "navigation",
+    "savedObjects",
+    "kibanaLegacy",
+    "spaces"
```

Comment:
The `spaces` plugin can be disabled in 7.16, so you might want to move this to `optionalPlugins`. Otherwise, disabling the `spaces` plugin would implicitly disable the `graph` plugin.